src/ode_solver/ode_solver_factory.cpp

```diff
@@ -125,6 +98,132 @@ void ODESolverFactory<dim,real,MeshType>::display_error_ode_solver_factory(Param
+        if (n_rk_stages == 1){
+            pcout << " with 1 stage..." << std::endl;
+            return std::make_shared<RungeKuttaODESolver<dim,real,1,MeshType>>(dg_input,rk_tableau);
+        }
+        if (n_rk_stages == 2){
+            pcout << " with 2 stages..." << std::endl;
+            return std::make_shared<RungeKuttaODESolver<dim,real,2,MeshType>>(dg_input,rk_tableau);
+        }
+        if (n_rk_stages == 3){
+            pcout << " with 3 stages..." << std::endl;
+            return std::make_shared<RungeKuttaODESolver<dim,real,3,MeshType>>(dg_input,rk_tableau);
+        }
+        if (n_rk_stages == 4){
+            pcout << " with 4 stages..." << std::endl;
+            return std::make_shared<RungeKuttaODESolver<dim,real,4,MeshType>>(dg_input,rk_tableau);
+        }
+        else{
+            pcout << "but number of stages was unclear. Aborting..." << std::endl;
+            std::abort();
+            return nullptr;
+        }
+    }
+    if (ode_solver_type == ODEEnum::rrk_explicit_solver){
+        if constexpr(dim==1){
+            const bool use_collocated_nodes = dg_input->all_parameters->use_collocated_nodes;
+            using PDEEnum = Parameters::AllParameters::PartialDifferentialEquation;
+            const PDEEnum pde_type = dg_input->all_parameters->pde_type;
+            const bool use_inviscid_burgers = (pde_type == PDEEnum::burgers_inviscid);
+            if (use_collocated_nodes && use_inviscid_burgers){
+                pcout << "Creating Relaxation Runge Kutta ODE Solver";
+                if (n_rk_stages == 1){
+                    pcout << " with 1 stage..." << std::endl;
+                    return std::make_shared<RRKExplicitODESolver<dim,real,1,MeshType>>(dg_input,rk_tableau);
+                }
+                if (n_rk_stages == 2){
+                    pcout << " with 2 stages..." << std::endl;
+                    return std::make_shared<RRKExplicitODESolver<dim,real,2,MeshType>>(dg_input,rk_tableau);
+                }
+                if (n_rk_stages == 3){
+                    pcout << " with 3 stages..." << std::endl;
+                    return std::make_shared<RRKExplicitODESolver<dim,real,3,MeshType>>(dg_input,rk_tableau);
+                }
+                if (n_rk_stages == 4){
+                    pcout << " with 4 stages..." << std::endl;
+                    return std::make_shared<RRKExplicitODESolver<dim,real,4,MeshType>>(dg_input,rk_tableau);
+                }
+                else{
+                    pcout << "but number of stages was unclear. Aborting..." << std::endl;
+                    std::abort();
+                    return nullptr;
+                }
+            } else {
+                pcout << "RRK has only been tested on collocated nodes with Burgers. Aborting..,"<<std::endl;
+                std::abort();
+            }
+        } else {
+            pcout << "RRK has only been tested on 1D calculations. Aborting..,"<<std::endl;
```

Comment:
```suggestion
            pcout << "\n Error: RRK has only been tested on 1D calculations. Aborting..."<<std::endl;
```

Reply:
I've added `Error` to all error messages, and fixed the accidental comma.